app/Http/Resources/Stats.php

```diff
@@ -0,0 +1,24 @@
+<?php
+
+namespace App\Http\Resources;
+
+use App\Contracts\Resource;
+
+/**
+ * @mixin \App\Models\Stats
+ */
+class Stats extends Resource
+{
+    public function toArray($request)
+    {
+        return [
+            'flights'         => $this->flights,
+            'total_time'      => $this->flight_time,
+            'average_time'    => $this->average_flight_time,
+            'average_score'   => number_format($this->average_score),
+            'balance'         => $this->balance,
+            'average_fuel'    => number_format($this->average_fuel_used / 2.20462262185).' Kg',
```

Comment:
I do not think that this is correct. It should be casted as Fuel and then we can get both lbs and kg results.

Forcing something to `kg` is not logical 😉

Reply:
Something like below should work
```php
use App\Support\Units\Fuel

'average_fuel' => Fuel::make($this->average_fuel_used, config('phpvms.internal_units.fuel'))->getResponseUnits();
```